package.json

```
@@ -45,15 +45,20 @@
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.1.0"
+    "@jupyterlab/application": "^3.1.0",
+    "@jupyterlab/docregistry": "^3.2.8",
+    "@lumino/commands": "^1.20.0",
+    "@jupyterlab/apputils": "^3.2.8",
```

You need to use consistent version for `@jupyterlab` package - otherwise expect troubles. And to be compatible with a maximal number of versions, I strongly advice you to pin the latest major or minor version:

```suggestion
    "@jupyterlab/application": "^3.0.0",
    "@jupyterlab/apputils": "^3.0.0",
    "@jupyterlab/docregistry": "^3.0.0",
    "@lumino/commands": "^1.20.0",
```